images/kubectl-build-deploy-dind/helmcharts/mongodb-dbaas/templates/prebackuppod.yaml

```diff
@@ -0,0 +1,67 @@
+{{ if .Capabilities.APIVersions.Has "backup.appuio.ch/v1alpha1/PreBackupPod" }}
+apiVersion: backup.appuio.ch/v1alpha1
+kind: PreBackupPod
+metadata:
+  name: {{ include "mongodb-dbaas.fullname" . }}-prebackuppod
+  labels:
+    {{- include "mongodb-dbaas.labels" . | nindent 4 }}
+  annotations:
+    {{- include "mongodb-dbaas.annotations" . | nindent 4 }}
+spec:
+  backupCommand: mongodump --uri=mongodb://${BACKUP_DB_USER}:${BACKUP_DB_PASSWORD}@${BACKUP_DB_HOST}:${BACKUP_DB_PORT}/${BACKUP_DB_NAME}?ssl=true&sslInsecure=true&tls=true&tlsInsecure=true --archive
```

Comment:
all the other backupCommands run their commands inside `/bin/sh -c`
I'm pretty sure we have to do it this way, so I suggest to do the same here :)